internal/test/scorecard/clients.go

```diff
@@ -158,9 +158,9 @@ func create[r runtime.Object](ctx context.Context, c rest.Interface, res string,
 	return result, err
 }
 
-func update[r runtime.Object](ctx context.Context, c rest.Interface, res string, ns string, obj r, result r) (r, error) {
+func update[r runtime.Object](ctx context.Context, c rest.Interface, res string, ns string, obj r, result r, name string) (r, error) {
 	err := c.Put().
-		Namespace(ns).Resource(res).
+		Namespace(ns).Resource(res).Name(name).
```

Comment:
Just curious here: The body is populated with the object, which should have the object name in metadata already right? Or am I misunderstanding...?

Reply:
When I didn't send the name directly into the request I would get an error about a missing name?

Reply:
Ahh i see thanks! I looked into deployment client and they do the same :))